# Задание 2

Два лучника стреляют в мишень.
Когда они независимо стреляют, тренируются, то первый из них попадает в мишень с вероятностью *70%*, а второй - *55%*. 
Теперь они выстрелили вместе, в результате в мишени торчит одна стрела.
Вопрос: какова вероятность, что **попал первый лучник**, а какова - что **попал второй**?

---
## Решение

Известно, что 
 - вероятность попадания первого лучника `p_arch1 = 0.70`;
 - вероятность попадания второго лучника `p_arch2 = 0.55`;

Следовательно, вероятность промаха по мишени у лучников равна:

In [1]:
p_arch1 = 0.70
p_arch2 = 0.55

q_arch1 = 1 - p_arch1
q_arch2 = 1 - p_arch2
print(f'- вероятность промаха первого лучника {q_arch1:0.3f}')
print(f'- вероятность промаха второго лучника {q_arch2:0.3f}')

- вероятность промаха первого лучника 0.300
- вероятность промаха второго лучника 0.450


Для оперирование событиями введём:
 - `A1` первый лучник попал в цель, т.е. `p(A1) = p_arch1`;
 - `A2` второй лучник попал в цель, т.е. `p(A2) = p_arch2`;
 - `not_A1` первый лучник промазал, т.е. `p(not_A1) = q_arch1`;
 - `not_A2` второй лучник промазал, т.е. `p(not_A2) = q_arch2`;
 
В задании обозначено событие `Y` - при двух выстрелах попадает только один из лучников.
Это возможно, когда:
 - первый лучник попал в цель и второй промазал то есть `A1 * not_A2` или `p_arch1 * q_arch2`
 - второй лучник попал в цель и первый промазал то есть `A2 * not_A1` или `p_arch2 * q_arch1`

In [2]:
print(f'Вероятность, что первый лучник попал в цель и второй промазал: {p_arch1 * q_arch2:0.3f}')
print(f'Вероятность, что второй лучник попал в цель и первый промазал: {p_arch2 * q_arch1:0.3f}')

Вероятность, что первый лучник попал в цель и второй промазал: 0.315
Вероятность, что второй лучник попал в цель и первый промазал: 0.165


Так как других вариантов для получения одного попадания нет, а эти два варианта - несовместные (они не могут произойти одновременно), то по теореме сложения вероятностей несовместных событий определим вероятность события `Y`

`p_Y = A1 * not_A2 * + not_A1 * A2`

или

`p_Y = p_arch1 * q_arch2 + q_arch1 * p_arch2`

In [3]:
p_Y = p_arch1 * q_arch2 + q_arch1 * p_arch2
print(f'Вероятность, что при двух выстрелах попадает только один из лучников: {p_Y:0.3f}')

Вероятность, что при двух выстрелах попадает только один из лучников: 0.480


## Ответ

Вероятность озвученного в задании события, что при двух выстрелах попадает только один из лучников: 0.480

Вероятность, что в мишени находится стрела выпущенная первым лучником: 0.315

Вероятность, что в мишени находится стрела второго лучника: 0.165
